students/Osiddiquee/lesson02/grid.py

```diff
@@ -0,0 +1,15 @@
+border = '+' + 4 * ' -' + ' +' + 4 * ' -' + ' +'
+interior = '|' + 9 * ' ' + '|' + 9 * ' ' + '|'
+
+
+print(border)
+print(interior)
+print(interior)
+print(interior)
+print(interior)
+print(border)
```

Comment:
Not the most DRY code, but you already have the loops idea in your other submissions for this, so I think you got the idea.